### PR TITLE
Add timeseries chrome and simple date picker

### PR DESCRIPTION
**Pull request by ranquild**
Demo https://www.loom.com/share/459896063732459e8e78e44d5361ea50
Relies on https://github.com/metabase/metabase/pull/35678

How to verify:
- New -> Orders -> Sum by Count, Group by Created At (Month)
- Visualize
- You should see the timeseries chrome at the bottom (View ... by ...)
- It should be possible to apply the filter on the breakout column and change the selected temporal bucket

Notes:
- The idea is to add a "simple" variant of `DatePicker`. It uses almost the same logic, however the UI is different. I tried different approaches and the one I liked most is creating a "Simple<>" component close to the existing component and re-using its utilities and subcomponents. It makes sure these 2 will be in sync.
- I'll add tests in another PR since this is already a huge PR.

Screenshot 2023-11-15 at 16 38 44
Screenshot 2023-11-15 at 16 38 50
Screenshot 2023-11-15 at 16 38 59

### CODE REVIEW

File: frontend/src/metabase/common/components/FilterPicker/DateFilterPicker/DateFilterPicker.tsx [hunk 1]
```diff
@@ -58,3 +61,45 @@ export function DateFilterPicker({
...
+  onChange: (filter: Lib.ExpressionClause | undefined) => void;
+}
+
+export function SimpleDateFilterPicker({
+  query,
+  stageIndex,
+  column,
+  filter,
+  onChange,
+}: SimpleDateFilterPickerProps) {
```
Comment (kulyk) [inline frontend/src/metabase/common/components/FilterPicker/DateFilterPicker/DateFilterPicker.tsx:79]: nit: I think it'd be more intuitive to have a `SimpleDateFilterPicker` file

File: frontend/src/metabase/common/components/DatePicker/DatePicker.tsx [hunk 2]
```diff
@@ -87,3 +90,35 @@ export function DatePicker({
...
+  value?: DatePickerValue;
+  availableOperators?: ReadonlyArray<DatePickerOperator>;
+  onChange: (value: DatePickerValue | undefined) => void;
+}
+
+export function SimpleDatePicker({
+  value: initialValue,
+  availableOperators = DATE_PICKER_OPERATORS,
+  onChange,
+}: SimpleDatePickerProps) {
```
Comment (kulyk) [inline frontend/src/metabase/common/components/DatePicker/DatePicker.tsx:104]: nit: I think it'd be more intuitive to have a `SimpleDatePicker` file

File: e2e/test/scenarios/question/reproductions/22247-timeseries-filter-all-time.cy.spec.js [hunk 3]
```diff
@@ -14,27 +14,25 @@ describe("time-series filter widget", () => {
...
-        .should("eq", "All Time");
+      cy.findByDisplayValue("All time").should("be.visible");
 
       cy.button("Apply").should("not.be.disabled");
     });
   });
 
   // Skip the rest of the tests until https://github.com/metabase/metabase/issues/22973 gets resolved
-  it.skip("should allow switching from All Time filter", () => {
+  it.skip("should allow switching from All time filter", () => {
```
Comment (kulyk) [inline e2e/test/scenarios/question/reproductions/22247-timeseries-filter-all-time.cy.spec.js:35]: Is there a chance we can un-skip this test?

File: frontend/src/metabase/visualizations/click-actions/components/TimeseriesFilterWidget.jsx
Comment (kulyk) [inline frontend/src/metabase/visualizations/click-actions/components/TimeseriesFilterWidget.jsx:1]: fire-elmo-gif

**Review by kulyk — APPROVED** [review]
LGTM!

File: frontend/src/metabase/common/components/FilterPicker/DateFilterPicker/DateFilterPicker.tsx (hunk 1 above)
Comment (ranquild) [reply frontend/src/metabase/common/components/FilterPicker/DateFilterPicker/DateFilterPicker.tsx:79]: Agree, fixed

File: frontend/src/metabase/common/components/DatePicker/DatePicker.tsx (hunk 2 above)
Comment (ranquild) [reply frontend/src/metabase/common/components/DatePicker/DatePicker.tsx:104]: Agree, fixed

**Pull request merged**